testing/utils/async_.py

```diff
@@ -0,0 +1,21 @@
+"""
```

Comment:
Why do you need this?

Reply:
@eugene-babichenko, if I mock async function result, result should be returned as like from `async function` because I `await` it. This is a trick to do this. See the function's references.